app/router.js

```diff
@@ -85,6 +85,7 @@ Router.map(function() {
           this.route('documents', { path: '/documenten', });
           this.route('requests', { path: '/aanvragen', });
         });
+        this.route('publications', { path: '/publicaties', });
```

Comment:
De route name is wat verwarrend omdat er een parent route is die ook al `publications` noemt. Misschien is `publication-activities` iets duidelijker om het onderscheid te maken?   
(Ik denk dat de meest juiste oplossing eigenlijk is om de root route te hernoemen naar `publication-flows`, maar dat gaat ons veel te ver leiden in deze PR)